webui/react/src/components/kit/Tooltip.tsx

```diff
@@ -32,11 +37,25 @@ const Tooltip: React.FC<TooltipProps> = ({
   placement = 'top',
   ...props
 }: TooltipProps) => {
+  if (isString(content)) {
```

Comment:
Not sure if using different styling behind the scenes based on the type of `content` is the *best* solution, but I went with it here as the least disruptive way to allow `Tooltip` to be used for both text content and component content (per our previous decision to consolidate `Popover` with `Tooltip`), while continuing to use antd's default styling. Also see updated DesignKit examples to see how these default styles differ.